src/resolve/resolved-dns-stub.c

```diff
@@ -89,15 +89,6 @@ DnsStubListenerExtra *dns_stub_listener_extra_free(DnsStubListenerExtra *p) {
         return mfree(p);
 }
 
-uint16_t dns_stub_listener_extra_port(DnsStubListenerExtra *p) {
-        assert(p);
-
-        if (p->port > 0)
-                return p->port;
-
-        return 53;
-}
```

Comment:
i wouldn't really bother with this too much. given lto the compiler should figure this out on its own, whether its worth inlining this

Reply:
It makes the code shorter and thus easier to follow.